brigade-worker/test/prestart.ts

```diff
@@ -2,6 +2,8 @@ import "mocha";
 import { assert } from "chai";
 import * as mock from "mock-require";
 import * as sinon from "sinon";
+import { getDefaultCompilerOptions } from "typescript";
```

Comment:
Is this import statement used?

Reply:
Oops, thank you; both this and the line below were unused.  Removed.